### ISSUE TITLE
How should we handle permissions for global search fields

### ISSUE THREAD

**Issue by maurizi** [OpenTreeMap/otm-core#2369]
In the advanced search pane we only show fields if the current user has at the viewing permission on the field.

The `address_street`, `address_city` and `address_zip` fields participate in the field permission system with field permissions on each of the base models (`Plot`, `RainBarrel`, etc.)

It would be nice to support searching on the address fields of all model types at once, but it is unclear how that interacts with permissions.

Some options I can think of:
 - Change the permission options of address fields so that they must always be visible (this is why we don't have issues with the global `updated_at` search).
 - Show a single address field if that field is visible for all mdoels on the map
 - Show a single address field if that field is visible for any model on the map
 - Show separate address fields for each model type, so instead of searching for "City" you would search for "Plot City" or "Bioswale City"

**Comment by dboyer** [comment]
Good catch. There are likely technical reasons to favor one of the suggested options over another, but from a non-tech perspective:

- I could see some cases where a map owner might not want all parts of the address field visible. I realize you can see this info by looking at the map showing the tree, but it might be a matter of removing unnecessary fields the map owner doesn't want cluttering up the detail page.

- This seems like a possibility.

- This also seems like a possibility.

- Searching for plot city vs bioswale city vs rain barrel city sounds like it would clutter the search UI so I'm not as excited about that idea.
